src/Symfony/Component/Inflector/Inflector.php

```diff
@@ -26,7 +26,6 @@
      * @var array
      *
      * @see http://english-zone.com/spelling/plurals.html
-     * @see http://www.scribd.com/doc/3271143/List-of-100-Irregular-Plural-Nouns-in-English
```

Comment:
Perhaps we can link to: http://grammar.about.com/od/words/a/A-List-Of-Irregular-Plural-Nouns-In-English.htm